week1-section3/exercise 8/Main.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+Exercise 8 - Easier Lottery
+Rewrite the previous lottery but now instead of getting $10 for each number matched in the exact same index, give the user $10 for any number that matches anywhere in the sequence.
+*/
+#include <iostream>
+
+#include <cstdlib>
+
+
+int main()
+{
+
+
+	std::cout << "Welcome to the lottery!" << std::endl;
+
```

Comment:
Is it really necessary to add all this empty space between the lines of your code?